src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/UpdateAcceleration.hpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "DataStructures/VariablesTag.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp"
+#include "Parallel/AlgorithmExecution.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "ParallelAlgorithms/Actions/MutateApply.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace CurvedScalarWave::Worldtube {
+
+/// @{
+/*!
+ * \brief Computes the geodesic acceleration of the particle, see
+ *`Tags::GeodesicAccelerationCompute`.
+ */
+struct UpdateAccelerationMutator {
```

Comment:
What component will this be run on? (the next commit answers this, but probably good to put it in the comment as well)

Reply:
added a comment, it is also in the `SingletonActions` folder though ;)

Reply:
Ah yes. That does help in the code, but not so much in the docs. Thanks for adding the comment!